lib/model/Capability.mjs

```diff
@@ -243,6 +206,27 @@ const namePerType = {
   Generic: cap => cap.comment || `Generic`
 };
 
+/**
+ * @param {object} cap The capability (with a set slot property) to generate a name for.
+ * @returns {string} The name for the capability, without the comment appended (if any).
+ */
+function getSlotCapabilityName(cap) {
+  let name;
+
+  if (cap.slot[0].number === cap.slot[1].number) {
+    name = cap.wheelSlot[0].name;
+  }
+  else {
+    name = cap.wheelSlot.map(slot => slot.name).join(` … `);
+  }
+
+  if (cap.colorTemperature) {
```

Comment:
This should be a property of `wheelSlot`. Maybe it'll be better to include it in the slot's name?

Reply:
The capability's `colorTemperature` getter uses the value from the wheel slot if there is one, so this works as is.

Didn't we agree on not using the comment feature in this PR, but rather in the issue?